pages/linux/nordvpn.md

```diff
@@ -19,6 +19,10 @@
 
 `nordvpn countries`
 
-- Connect to a NordVPN server in a specific country:
+- Connect to a NordVPN server in a specific country and city (optionally):
```

Comment:
We have space in this page, we can split this into 2 examples.

Reply:
please, review